src/components/nodes/startEvent/startEvent.vue

```diff
@@ -63,6 +63,11 @@ export default {
           nodeType: 'processmaker-modeler-message-start-event',
           dataTest: 'switch-to-message-start-event',
         },
+        {
```

Comment:
I put Boundary Signal Event above Message Event - since it can be disabled. Should we bump this one above too?

Reply:
Yeah, I'll move it up!